curriculum/challenges/english/25-front-end-development/quiz-javascript-math/66edc3ab8c6413c344f401bf.md

```diff
@@ -17,439 +17,475 @@ Answer all of the questions below correctly to pass the quiz.
+`Math.rand()`
 
 ---
 
-Placeholder distractor 3
+`Math.random(1,10)`
 
 #### --answer--
 
-Placeholder answer
+`Math.random()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following will return `false`?
 
 #### --distractors--
 
-Placeholder distractor 1
+`isNaN("1998", "1999")`
 
 ---
 
-Placeholder distractor 2
+`isNaN({})`
 
 ---
 
-Placeholder distractor 3
+`isNaN(undefined)`
 
 #### --answer--
 
-Placeholder answer
+`isNaN(null)`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following will convert `172` to its hexadecimal equivalent?
 
 #### --distractors--
 
-Placeholder distractor 1
+`(-0xa5).toString(2);`
 
 ---
 
-Placeholder distractor 2
+`(172).string(0x);`
 
 ---
 
-Placeholder distractor 3
+`(172).parseString(0x);`
 
 #### --answer--
 
-Placeholder answer
+`(172).toString(16);`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is the right way to convert a string to an integer?
 
 #### --distractors--
 
-Placeholder distractor 1
+`parseINT("300")`
 
 ---
 
-Placeholder distractor 2
+`int("300")`
 
 ---
 
-Placeholder distractor 3
+`integer("300")`
 
 #### --answer--
 
-Placeholder answer
+`parseInt("300")`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following statements is false?
 
 #### --distractors--
 
-Placeholder distractor 1
+`parseFloat()` ignores trailing invalid characters
 
 ---
 
-Placeholder distractor 2
+`parseFloat()` can parse a string that begins with `Infinity` or `-Infinity`
 
 ---
 
-Placeholder distractor 3
+`parseFloat("0x1f")` will return 0
 
 #### --answer--
 
-Placeholder answer
+`parseFloat()` requires two parameters
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is false?
 
 #### --distractors--
 
-Placeholder distractor 1
+A number greater than 100 as an argument to the `Number.prototype.toFixed()` method will result to a range error
 
 ---
 
-Placeholder distractor 2
+`Number.prototype.toFixed(numDecimalPlaces)` is used to return a number converted to a string with decimals places depending on the argument
 
 ---
 
-Placeholder distractor 3
+Calling the `Number.prototype.toFixed()` method on a number object with decimal points and no argument will round the number to the nearest ones
 
 #### --answer--
 
-Placeholder answer
+Using `Number.prototype.toFixed()` without an argument will default add a decimal point followed by a zero
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is not a comparison operator?
 
 #### --distractors--
 
-Placeholder distractor 1
+`!=`
 
 ---
 
-Placeholder distractor 2
+`>=`
 
 ---
 
-Placeholder distractor 3
+`===`
 
 #### --answer--
 
-Placeholder answer
+`!>`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is false?
 
 #### --distractors--
 
-Placeholder distractor 1
+Exponentiation has a higher operator precedence than grouping
```

Comment:
```suggestion
Exponentiation has a lower operator precedence than grouping
```
To be honest I initially wasn't entirely sure what _grouping_ meant in this context.

Reply:
i should have said grouping operator ()

Reply:
This change got lost once again in the meantime. 😛